### PR TITLE
[TAS-209] ✨ Support querying recent creator counts

### PR DESCRIPTION
**Pull request by WeiJun0827**

### CODE REVIEW

File: db/stats.go [hunk 1]
```diff
@@ -78,6 +79,47 @@ func GetNftCreatorCount(conn *pgxpool.Conn) (count QueryCountResponse, err error
...
+	SELECT 
+		(SELECT COUNT(DISTINCT rc.sender)
+			FROM recent_creators AS rc
+			LEFT JOIN early_creators AS ec 
+				ON rc.sender = ec.sender
+			WHERE ec.sender IS NULL) AS new_creator_count,
+		(SELECT COUNT(DISTINCT rc.sender)
+			FROM recent_creators AS rc
+			JOIN early_creators AS ec 
+				ON rc.sender = ec.sender) AS returning_creator_count;
```
Comment (williamchong) [inline db/stats.go:110]: Seems we only need to know about address that exists in `recent_creators`, but `early_creators` query the whole table
Would we do better with LATERAL query than self join (or sth)?

Comment (WeiJun0827) [reply db/stats.go]: Per Edmond's use case, I return the all-time result with another query.
See if it's OK:
https://node.testnet.like.co/statistics/nft/returning-creator-count?period_days=30

Comment (williamchong) [reply db/stats.go]: Nice to use window function, but I dont think we should hardcode all data in weeks interval
Just use  (period_days) as window interval, return latest 1 entry by default, and accept a start/end time as param if really needed?

Comment (williamchong) [reply db/stats.go]: Say for period_days=30, we should use return the data for "now-30days to now"

**Review by williamchong — APPROVED** [review]

**Pull request merged**